agent/command/shell.go

```diff
@@ -69,6 +72,11 @@ type shellExec struct {
 	// allows following commands to execute even if this shell command fails.
 	ContinueOnError bool `mapstructure:"continue_on_err"`
 
+	// Debug is a temporary and deliberately undocumented feature added in EVG-8017
+	// and should be removed. It writes a file called debug_shell_exec.<id>.yml that
+	// contains the contents of the command and its expansions.
+	Debug bool `map:"debug"`
```

Comment:
Does this tag need mapstructure as opposed to map?

Reply:
Ha, thanks, fixed. Weirdly I tested this end-to-end in staging, and it worked.

Reply:
Yeah for this the default mapstructure tag would still be `debug`, but